spec/models/job_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Job, type: :model do
+  it { should validate_presence_of(:title) }
+  it { should validate_presence_of(:description) }
+  it { should validate_presence_of(:job_type) }
+
+  describe 'company creation specs' do
+    it 'should be able to save job only with a user who the ocmpany belongs to' do
+      user = build(:user)
+      user.save!
```

Comment:
you can avoid `.save` commands here by calling `create` instead of `build` in the line above